util/src/services/ChannelService.ts

```diff
@@ -0,0 +1,88 @@
+import { Channel, ChannelType, PublicUserProjection, Recipient, User } from "../entities";
+import { HTTPError } from "lambert-server";
+import { emitEvent, trimSpecial } from "../util";
+import { DmChannelDTO } from "../dtos";
+import { ChannelRecipientRemoveEvent } from "../interfaces";
+
+export function checker(arr: any[], target: any[]) {
+	return target.every(v => arr.includes(v));
+}
+
+export class ChannelService {
```

Comment:
Instead of creating a seperate ChannelService you could integrate it directly into the ``Channel`` Entity class.

Reply:
Whouldn't be better to keep DB's Entity definitions and functions separated? It's usually done like that in frameworks like Spring Boot and i've also seen it done in other express/typeorm projects and tutorial like https://github.com/bymi15/express-typeorm-rest-boilerplate/tree/main/src/api